act/signed-commit/action.yml

```diff
@@ -29,30 +23,77 @@ inputs:
     required: false
     default: "."
 
-  github-token:
-    description: >-
-      Github token used for interacting with Actions API. Must have Contents:Write permissions
-    required: true
+outputs:
+  commit-sha:
+    description: SHA of the created commit
+  commit-url:
+    description: URL of the commit on GitHub
 
 runs:
   using: composite
   steps:
-  - name: Commit changes
-    shell: bash
-    working-directory: ${{ inputs.repo-path }}
-    env:
-      GH_TOKEN: ${{ inputs.github-token }}
-      BRANCH: ${{ inputs.branch }}
-      FILE: ${{ inputs.file }}
-      REPO: ${{ inputs.repo || github.repository }}
-      MESSAGE: ${{ inputs.message }}
-    run: |
-      export SHA=$(git rev-parse "$BRANCH:$FILE")
-      export CONTENT=$(base64 -i "$FILE")
-
-      gh api --method PUT "/repos/$REPO/contents/$FILE" \
-          --field message="$MESSAGE" \
-          --field content="$CONTENT" \
-          --field encoding="base64" \
-          --field branch="$BRANCH" \
-          --field sha="$SHA"
+
+    - name: Import PGP key
+      shell: bash
+      env:
+        PGP_KEY_B64: ${{ secrets.FPF_BRANCH_UPDATER_PGP_KEY }}
+      run: |
+        if [[ -z "$PGP_KEY_B64" ]]; then
+          echo "Missing required secret: FPF_BRANCH_UPDATER_PGP_KEY"
+          exit 1
+        fi
+
+        echo "$PGP_KEY_B64" | base64 --decode | gpg --import
+
+        KEY_ID=$(
```

Comment:
Is this env var meant to be output for usage in later steps? I think this is just living in the current job env at the moment